From 4ee0bd3c8ead89541ab7d196fb54e940e397420d Mon Sep 17 00:00:00 2001
From: Oliver Kurth <okurth@vmware.com>
Date: Fri, 7 Feb 2020 10:39:38 -0800
Subject: [PATCH] Rectify a log spew in vmsvc logging (vmware-vmsvc-root.log )

When a LSI Logic Parallel SCSI controller sits in PCI bus 0
(SCSI controller 0), the Linux disk device enumeration does not provide
a "label" file with the controller name.  This results in messages like

"GuestInfoGetDiskDevice: Missing disk device name; VMDK mapping unavailable for "/var/log", fsName: "/dev/sda2"

repeatedly appearing in the vmsvc logging.  The patch converts what previously
was a warning message to a debug message and thus avoids the log spew.
---
 open-vm-tools/services/plugins/guestInfo/diskInfo.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: pkg-open-vm-tools/open-vm-tools/services/plugins/guestInfo/diskInfo.c
===================================================================
--- pkg-open-vm-tools.orig/open-vm-tools/services/plugins/guestInfo/diskInfo.c
+++ pkg-open-vm-tools/open-vm-tools/services/plugins/guestInfo/diskInfo.c
@@ -846,9 +846,9 @@ GuestInfoGetDiskDevice(const char *fsNam
     */
    for (indx = 0; indx < partEntry->diskDevCnt; indx++) {
       if (partEntry->diskDevNames[indx][0] == '\0') {
-         g_warning("%s: Missing disk device name; VMDK mapping unavailable "
-                   "for \"%s\", fsName: \"%s\"\n", __FUNCTION__,
-                   partEntry->name, fsName);
+         g_debug("%s: Missing disk device name; VMDK mapping unavailable "
+                 "for \"%s\", fsName: \"%s\"\n", __FUNCTION__,
+                 partEntry->name, fsName);
          partEntry->diskDevCnt = 0;
          free(partEntry->diskDevNames);
          partEntry->diskDevNames = NULL;
